index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-answer-screen">0</div>
```

Comment:
That was something I did in my last iteration of this project. Just to clarify, an `answer-screen` div which is inside of some other div so that there's a separation between the actual thing holding the screen and the thing that I might be applying JavaScript logic to?